frontend/src/pages/Common/DatePicker.js

```diff
@@ -1,14 +1,17 @@
 import React from "react";
 import TextField from "@material-ui/core/TextField";
 import { withStyles } from "@material-ui/core";
+import { convertDatetimeToDate } from "../../helper";
 
 const styles = {
   form: {
     marginTop: "12px"
   }
 };
 
-function DatePicker({ classes, name, label, value = "", onChange, id = "default" }) {
+function DatePicker({ classes, name, label, onChange, value = "", id = "default" }) {
+  // Remove time to show the value properly in the DatePicker
+  value = convertDatetimeToDate(value);
```

Comment:
I suggest renaming value to datetime so it's more clear where the time in the comment comes from and using dayjs(datetime).format("YYYY-MM-DD") because the convertDatetimeToDate function pumps up the helper.js, it is only used in the DatePicker and without function it's still a one liner

Reply:
done